docs/examples/tutorial/cdef_classes/sin_of_square.py

```diff
@@ -0,0 +1,16 @@
+if cython.compiled:
+    from cython.cimports.libc.math import sin
+else:
+    from math import sin
```

Comment:
This isn't needed (and not relevant in the context of this example). There is a fallback for `libc.math` in the form of Python's own `math` module.
```suggestion
from cython.cimports.libc.math import sin
```

Reply:
> There is a fallback for `libc.math` in the form of Python's own `math` module.

Oh, I didn't know, I had to look at the source code first ..